index.html

```diff
@@ -53,5 +53,158 @@ <h2 class="job">I'm a software developer</h2>
         <a href="#"><img src="./images/Disabled.svg" alt="next_button" /></a>
       </div>
     </section>
+    <section class="works">
+      <a class="title" href="#">My Recent Works</a>
+      <hr />
+      <div class="cards">
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
```

Comment:
- [ ] The list of icons has a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). **Please apply it to other lists also.**